### PR TITLE
Hyperlinked the thumbor logo in readme.md

### PR DESCRIPTION
**Pull request by Abubakrce19**
I Hyperlinked the thumbor logo with its official website in readme.md file 
so the user can visit website with single click. 
Minor but impactfull.
Thank you

### CODE REVIEW

**Comment by Abubakrce19** [comment]
@sunny 
@Bertg 
@robolson 
@rfloriano

File: README.mkd [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 <p align="center">
   <!-- <img title="thumbor" alt="thumbor" src="https://raw.github.com/thumbor/thumbor/master/logo-thumbor.png" /> -->
+  <a href="http://www.thumbor.org">
   <img title="thumbor" alt="thumbor" src="https://github.com/thumbor/thumbor/blob/readme/docs/thumbor-logo.png?raw=true" />
```
Comment (sunny) [inline README.mkd:6]:
```suggestion
    <img title="thumbor" alt="thumbor" src="https://github.com/thumbor/thumbor/blob/readme/docs/thumbor-logo.png?raw=true" />
```

File: README.mkd [hunk 2]
```diff
@@ -2,7 +2,9 @@
 
 <p align="center">
   <!-- <img title="thumbor" alt="thumbor" src="https://raw.github.com/thumbor/thumbor/master/logo-thumbor.png" /> -->
```
Comment (sunny) [inline README.mkd:4]: Might as well clean up while we’re here.

```suggestion
```

**Comment by Abubakrce19** [comment]
@sunny 
Done

**Comment by Abubakrce19** [comment]
When will i get my swags for contributing to globo in hacktoberfest?
@heynemann 
@coveralls

**Comment by Abubakrce19** [comment]
image
I am top on leaderboard and haven't received
 the t shirt yet.

**Pull request merged**